src/test/java/net/openhft/chronicle/core/CoreTestCommon.java

```diff
@@ -1,25 +1,19 @@
 package net.openhft.chronicle.core;
 
+import net.openhft.chronicle.core.internal.JvmExceptionTracker;
 import net.openhft.chronicle.core.io.AbstractReferenceCounted;
-import net.openhft.chronicle.core.onoes.ExceptionKey;
-import net.openhft.chronicle.core.onoes.Slf4jExceptionHandler;
 import net.openhft.chronicle.core.threads.CleaningThread;
 import net.openhft.chronicle.core.threads.ThreadDump;
+import net.openhft.chronicle.testframework.internal.ExceptionTracker;
 import org.junit.After;
-import org.junit.Assert;
 import org.junit.Before;
 
-import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.function.Predicate;
-
 import static net.openhft.chronicle.core.io.AbstractCloseable.waitForCloseablesToClose;
 import static net.openhft.chronicle.core.io.AbstractReferenceCounted.assertReferencesReleased;
 
 public class CoreTestCommon {
```

Comment:
I like this very much - this boilerplate is duplicated everywhere (sometimes slightly differently)

It's a bummer that ExceptionKey cannot be easily moved to test-framework. While *we* only use it for testing, I assume @peter-k-lawrey made it available for people to use in production code. @peter-k-lawrey is it worth reviewing this?

Reply:
I agree @JerryShea . It would be highly desirable to move it to the test framework.